src/main/java/com/xjeffrose/xio/server/XioServerConfig.java

```diff
@@ -5,112 +5,40 @@
 import com.xjeffrose.xio.SSL.TlsConfig;
 import io.netty.channel.ChannelOption;
 import io.netty.util.Timer;
+import lombok.Getter;
 
 import java.util.Map;
 import java.util.concurrent.ExecutorService;
 import java.net.InetSocketAddress;
 
 public class XioServerConfig {
-  // old
+  @Getter
```

Comment:
I'm being pedantic here, someone who isn't familiar with lombok might be surprised by the class level @Getter